addons/l10n_mx/data/account_report_diot.xml

```diff
@@ -4,7 +4,6 @@
         <field name="name">DIOT</field>
         <field name="name@es_419">DIOT</field>
         <field name="root_report_id" ref="account.generic_tax_report"/>
-        <field name="filter_show_draft" eval="False"/>
```

Comment:
Same comment as on enterprise; just putting it here as well to make sure you don't forget :)